svf/lib/SABER/SaberSVFGBuilder.cpp

```diff
@@ -276,6 +276,8 @@ void SaberSVFGBuilder::rmIncomingEdgeForSUStore(BVDataPTAImpl* pta)
                     }
                     for (SVFGEdge* edge: toRemove)
                     {
+                        if (isa<StoreSVFGNode>(edge->getSrcNode()))
```

Comment:
Do we need this condition? Not all edges in toRemove?

Reply:
Yes, the condition should be removed. But [this case](https://github.com/SVF-tools/Test-Suite/blob/master/src/mem_leak/malloc43.c) won't pass because of an IntraMSSAPhinode to strong update. I'm afraid there will be more false positives.